documentation/player_traits.txt

```diff
@@ -0,0 +1,15 @@
+Trait names are hardcoded in the DATs, these are some unimplemented ones found while looking for others
+ID  Name
+129	damage limit +
+130	animus solace
+131	animus misery
+132	con anima
+133	con brio
+121	inspiration
+122	sleight of sword
+124	superior
+125	magic acc bonus
+126	magic eva bonus
+104	true shot
+135	max hp boost II
+136	shield barrier
```

Comment:
Offset by 1536 from ROM\181\72.dat (so 1537 = 1)

[FFXI full trait list as of Janurary 11th 2024.csv](https://github.com/LandSandBoat/server/files/13907747/FFXI.full.trait.list.as.of.Janurary.11th.2024.csv)

Reply:
I've been auto generating our enum for years btw. Just haven't kept up recently.

Reply:
omg so i didn't have to guess and check

Reply:
![image](https://github.com/LandSandBoat/server/assets/131182600/0413decb-43bc-4c20-8640-6be0cacec342)

wow literally the last two traits eh

Reply:
added to the documentation txt in case people want to implement other missing traits